src/pages/library/dataBrowser-utils.js

```diff
@@ -122,15 +127,3 @@ export const useDataCatalog = () => {
   })
   return { dataCatalog, refresh, loading }
 }
-
-export const importDataToWorkspace = datasets => {
```

Comment:
I moved this into the using components and changed it to a single dataset on the principle of keeping it as close to the working code as possible - it isn't used elsewhere anymore.